### PR TITLE
Endonyms Submod

### PR DESCRIPTION
**Pull request by emprahtheodosius**

### CODE REVIEW

File: TGC/history/countries/LBN - Lebanon.txt [hunk 1]
```diff
@@ -1,7 +1,9 @@
 capital = 913
-primary_culture = mashriqi
-culture = bedouin
-religion = sunni
+primary_culture = druzes
+culture = syriac
+culture = druzes
```
Comment (anzu-3278) [inline TGC/history/countries/LBN - Lebanon.txt:4]: Both primary and accepted?

File: TGC/decisions/Colonisation-Asia.txt [hunk 2]
```diff
@@ -312,12 +312,12 @@ political_decisions = {
 	organize_lebanon = {
 		picture = map_arabia
 		potential = {
+			civilized = yes
+			NOT = { has_global_flag = lebanon_organized }
```
Comment (anzu-3278) [inline TGC/decisions/Colonisation-Asia.txt:316]: Any reason to put this back?

Comment (emprahtheodosius) [reply TGC/decisions/Colonisation-Asia.txt:316]: Nope, just a difference from copy/pasting from a diff change 1-2 days ago on main rep

File: TGC/history/countries/LBN - Lebanon.txt (hunk 1 above)
Comment (emprahtheodosius) [reply TGC/history/countries/LBN - Lebanon.txt:4]: On account of when different colonizers/GPs reorganize Lebanon, they will also be changing primary culture, ergo there should be accepted culture status for the group that fell out of grace

eg. French change Druze => to Syriac => but then Turks regain it and change to Mashriqi

technically just for flavour, most groups in LBN should be accepting each other

Comment (anzu-3278) [reply TGC/history/countries/LBN - Lebanon.txt:4]: I'm not sure that's gonna work but we'll see.

**Pull request merged**